### DISCUSSION TITLE
Can I start a list for tickets/issues that perhaps should be closed or discussed?

### DISCUSSION THREAD

**Discussion by derwin12** [xLightsSequencer/xLights#4303] [General]
Let me start #4277, #4056, #3030, #4255, #4093, #3873.
#2488, #1999 #2088 #2094 #2112 #2147
Now there are many older ones that can likely be closed. As I come across them I will post here.

**Comment by cybercop23** [comment]
#4175, #4166, #4163, #4160, #4141, #4140, #4136, #4130, #4120, #4129, #4128

**Reply by cybercop23** [reply]
#4136 only open

**Comment by cybercop23** [comment]
#4117, #4112, #1639, #4098, #4091, #4083, #4081, #4064, #4063, #4061, #4060, #4042, #4040, #4039, #4000


And then there's the dreeded #4016

**Comment by cybercop23** [comment]
#3986, #3982, #3948, #3944, #3941, #3929, #3919, #3907, #3892, #3879, #3863, #3862, #3858, #3857, #3855, #3847, #3838, #3829, #3823, #3818, #3807, #3800

@dkulp @keithsw1111 @computergeek1507 ... @derwin12 and I been going through thr open items (as I'm sure you know by know based on all of the notification emails) and these are issues that are either resolved, workarounds used or even fixed. Since we can't close them, maybe you guys can take a look and if the owner hasn't close them (we poked and asked them to close), can you please close them?

**Comment by cybercop23** [comment]
#3788, #3769, #3764, #3739, #3699, #3683, #3682, #3677, #3676, #3675, #3669, #3661, #3653, #3650, #3648, #3645, #3644, #3643, #3639, #3636, #3634, #3625, #3624, #3623, #3618, #3608, #3607, #3605, #3601

**Comment by cybercop23** [comment]
#3596, #3590, #3585, #3582, #3579, #3577, #3557, #3553, #3540, #3509, #3488, #3477, #3471, #3469, #3467, #3466, #3462, #3445, #3418

**Comment by cybercop23** [comment]
#3391, #3385, *** #3380, #3357, #3354, #3348, #3318, #3317, #3313, #3311, #3303, #3278, #3275, #3254, #3253, #3238, #3233, #3226

**Comment by cybercop23** [comment]
#3197, #3188, #3180, #3167, #3157, #3149, #3145, #3144, #3132, #3126, #3103,

**Comment by cybercop23** [comment]
#3101, #3092, #3005, #3003, #3001, #2972, #2964, #2954, #2952, #2949, #2946, #2939, #2925, #2904,
